test/http-server-test.js

```diff
@@ -154,5 +154,45 @@ vows.describe('http-server').addBatch({
         assert.ok(res.headers['access-control-allow-headers'].split(/\s*,\s*/g).indexOf('X-Test') >= 0, 204);
       }
     }
+  },
+  'When baseDir is specified': {
+    topic: function () {
+      var server = httpServer.createServer({
+        root: root,
+        baseDir: '/test'
+      });
+      server.listen(8083);
```

Comment:
After merging master and fixing merge conflicts, this port needs to change. I suggest 8086.

The URLs below will also need to be updated.